### ISSUE TITLE
Make it clear this doesn't provide normal browser cookie behavior

### ISSUE THREAD

**Issue by transitive-bullshit** [hstove/electron-cookies#12]
Normally, the browser will send cookies automatically based on the request domain and set cookies based on response set-cookie headers.

It'd be nice to add something to the README explaining that this behavior is not captured by this shim.

**Comment by hstove** [comment]
Agreed. If you want to add something to the README, I'd happily merge it.

**Comment by transitive-bullshit** [comment]
Let me know if you'd like to change any of the specifics -- thanks :)
